src/aui-layout/assets/aui-layout-builder-resize-col-core.css

```diff
@@ -42,6 +42,7 @@
     position: absolute;
     top: 0;
     width: 0;
+    z-index: 2;
 }
 
 .layout-builder-resize-col-breakpoint-line {
```

Comment:
Fix the z-index of the draggable elements provoked by the position absolute of the add field button.